packages/plans-grid-next/src/style.scss

```diff
@@ -414,22 +414,31 @@ $mobile-card-max-width: 440px;
 }
 
 .components-custom-select-control {
+	line-height: 0px;
```

Comment:
@jeyip This line overrides the global `components-custom-select-control` style on the page. Same for a few more additions added in some other PR that target this general class. should be addressed in https://github.com/Automattic/wp-calypso/pull/87382

(I had initially added these styles at the root level in one of my PRs. there were only localized styles enclosed at the time. this now should override things at other places)

Reply:
Thanks for the ping. Taking a look now 👀

Reply:
Ah I see -- I appreciate the heads up and the PR 🙏 Doing a quick smoke test of the the linked pull request